kubernetes_platform/proto/kubernetes_executor_config.proto

```diff
@@ -26,6 +26,8 @@ message KubernetesExecutorConfig {
     repeated PvcMount pvc_mount = 3;
     NodeSelector node_selector = 4;
     PodMetadata pod_metadata = 5;
+    // One of Always, Never, IfNotPresent.
+    string image_pull_policy = 6;

```

Comment:
nit: if this is a stable value set, maybe we can make this an enum instead of string? (I don't have strong opinion on this, feel free to ignore)

Reply:
@chensun  I think I'll leave it as a string if that's ok. Also, it seems that in [Kubernetes](https://github.com/kubernetes/kubernetes/blob/11b974043604f5ccbeb6e5e62e1d9edcf00bc336/staging/src/k8s.io/api/core/v1/generated.proto#L863) it is defined as a string as well, but I am not sure if it's the same case. Anyway, in the second part of the [PR ](https://github.com/kubeflow/pipelines/pull/10417) there is a check for the values. Thanks